app/src/main/res/values/styles.xml

```diff
@@ -1,18 +1,212 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+
     <style name="darkButtonStyle">
         <item name="android:background">@drawable/dark_background</item>
         <item name="android:fontFamily">@font/poppins_regular</item>
         <item name="android:textColor">@color/white</item>
-        <item name="android:textSize">17sp</item>
+        <item name="android:textSize">@dimen/_16sp</item>
         <item name="android:textAllCaps">false</item>
     </style>
 
     <style name="lightButtonStyle">
         <item name="android:background">@drawable/white_background</item>
         <item name="android:fontFamily">@font/poppins_regular</item>
         <item name="android:textColor">@color/black</item>
-        <item name="android:textSize">17sp</item>
+        <item name="android:textSize">@dimen/_16sp</item>
         <item name="android:textAllCaps">false</item>
     </style>
+
+    <style name="editTextLayoutStyleInfo">
+        <item name="android:layout_width">match_parent</item>
```

Comment:
width, height, margin, padding bunları style'a eklemiyoruz. İlgili view xml içinde verebiliriz.
textColor, textSize, backgroundColor, fontFamily gibi property'leri style'a çıkarabilirsin.

Ayrıca her view için style oluşturmaya da gerek yok. Buraya yazmak, ilgili xml içinde yazmaktan daha çok iş yükü çıkarır.

Reply:
Tamamdır düzeltiyorum hemen o tarafları